pyproject.toml

```diff
@@ -132,6 +133,12 @@ full = [
 [project.scripts]
 ralph = "ralph.__main__:cli.cli"
 
+[project.entry-points."ralph.backends.data"]
+ralph_data_backends = "ralph.backends.loader:DATA_BACKENDS"
```

Comment:
Should an entry point contain a list of dotted backend paths or just a single backend? WDYT?

Reply:
I prefer listing them explicitly. Your comment's proposal looks great to me.